gettsim/social_insurance.py

```diff
@@ -26,185 +24,193 @@ def soc_ins_contrib(person, tb):
     # a couple of definitions
 
     # As there is only one household, we selcet west_ost dependent paramter in the
-    # beginning and place them in a seperate dictionary tb_ost.
+    # beginning and place them in a seperate dictionary params_ost.
     westost = "o" if person["east"] else "w"
-    tb_ost = {}
+    params_ost = {}
```

Comment:
This construct is hard to understand. Why call that `params_ost` if it is west or east???

Reply:
It saves east and west specific variables. I could either store a new value in the dictionary, change the name of the dictionary, just take westost with me or do the check every time!

Reply:
Just leave for now and make an issue out of it that we should find a general treatment for east/west differences. Probably store these parameters always in arrays. Also see my comment on #75 here: https://gettsim.zulipchat.com/#narrow/stream/212222-general/topic/Plan.20until.20Christmas/near/181733445

Reply:
Opened #115